tests/app/main/views/test_email_branding.py

```diff
@@ -16,7 +16,7 @@ def test_email_branding_page_shows_full_branding_list(client_request, platform_a
     client_request.login(platform_admin_user)
     page = client_request.get(".email_branding")
 
-    links = page.select(".message-name a")
+    links = page.select(".email-brand a")
```

Comment:
Think you can do this then you don’t need to introduce an extra class
```suggestion
    links = page.select(".browse-list-item a")
```

Reply:
it was like this already so the live search had a class to update, however, we're already using generic selectors for live search in plenty of other pages so i've updated in 96ed2a7e6